values-schema.yaml

```diff
@@ -138,17 +138,17 @@ definitions:
       region:
         title: AWS region
         description: An AWS region.
-        type: string
+        $ref: '#/definitions/wordCharacterPattern'
```

Comment:
why not add the enum of AWS here?

Reply:
And maybe add all the enums for all the cloud providers, and then choose a `oneOf` for `cluster.region`